src/state/internal/createQueryStore.ts

```diff
@@ -474,7 +474,7 @@ export function createQueryStore<
                   )}`
                 );
               } else {
-                console.log('[🔄 Fetching 🔄] for params:', JSON.stringify(effectiveParams));
+                console.log('[🔄 Fetching 🔄] for params:', serializeParamsForLogging(effectiveParams));
```

Comment:
any reason not to use logger.info?

Reply:
these logs are only enabled if `debugMode` is in place.